R/species.r

```diff
@@ -0,0 +1,114 @@
+#' Infer Species
+#'
+#' @param sset a \code{SigSet}
+#' @param df_as a data.frame of alignment score for each probe.
+#' @param infer.human whether to use the fraction of negative probes to infer human.
+#' @param topN Top n positive and negative probes used to infer species.
+#' @return a list of auc, pvalue, species (NCBI official species names) and taxid.
+#' We infer species based on probes pvalues and alignment score.
+#' AUC was calculated for each specie, y_true is 1 or 0 
+#' for pval < 0.01 or pval > 0.2, respeceively,
+#' y_pred is normalized alignment score (AS / 50).
+#'
+#' Our function works on a single sample.
+#' @importFrom PRROC roc.curve
```

Comment:
remove PRROC call. use some custom code like 
```
 labels <- as.logical(labels)
    n1 <- sum(labels)
    n2 <- sum(!labels)
    R1 <- sum(rank(scores)[labels])
    U1 <- R1 - n1 * (n1 + 1)/2
    U1/(n1 * n2)
```

Reply:
It has been modified.